### ISSUE TITLE
Can't build docs

### ISSUE THREAD

**Issue by mrjoes** [flask-admin/flask-admin#1021]
@petrus-jvrensburg Something happened recently and readthedocs.org chokes on our docs.

Error looks like this:
![image](https://cloud.githubusercontent.com/assets/179880/9476620/97bba7d8-4b3b-11e5-9698-ed24d13ef5e2.png)

Here's build log:

```
Running Sphinx v1.3.1
making output directory...
loading translations [en]... done
loading pickled environment... not yet created
loading intersphinx inventory from http://docs.python.org/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [latex]: all documents
updating environment: 22 added, 0 changed, 0 removed
reading sources... [  4%] adding_a_new_model_backend
reading sources... [  9%] advanced
reading sources... [ 13%] api/index
reading sources... [ 18%] api/mod_actions
reading sources... [ 22%] api/mod_base
reading sources... [ 27%] api/mod_contrib_fileadmin
reading sources... [ 31%] api/mod_contrib_mongoengine
reading sources... [ 36%] api/mod_contrib_mongoengine_fields
reading sources... [ 40%] api/mod_contrib_peewee
reading sources... [ 45%] api/mod_contrib_pymongo
reading sources... [ 50%] api/mod_contrib_sqla
reading sources... [ 54%] api/mod_form
reading sources... [ 59%] api/mod_form_fields
reading sources... [ 63%] api/mod_form_rules
reading sources... [ 68%] api/mod_form_upload
reading sources... [ 72%] api/mod_helpers
reading sources... [ 77%] api/mod_model
reading sources... [ 81%] api/mod_model_template
reading sources... [ 86%] api/mod_tools
reading sources... [ 90%] changelog
reading sources... [ 95%] index
reading sources... [100%] introduction

/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:9: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:50: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:73: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:92: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:157: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:221: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:280: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:422: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:479: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/advanced.rst:498: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/api/mod_form_fields.rst:2: WARNING: Duplicate explicit target name: "select2".
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/index.rst:42: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/index.rst:49: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/introduction.rst:9: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/introduction.rst:96: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/introduction.rst:179: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/introduction.rst:305: ERROR: Document or section may not begin with a transition.
/home/docs/checkouts/readthedocs.org/user_builds/flask-admin/checkouts/v1.3.0/doc/introduction.rst:363: ERROR: Document or section may not begin with a transition.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
processing flask-admin... index introduction advanced adding_a_new_model_backend api/index api/mod_base api/mod_helpers api/mod_model api/mod_form api/mod_form_rules api/mod_form_fields api/mod_form_upload api/mod_tools api/mod_actions api/mod_contrib_sqla api/mod_contrib_mongoengine api/mod_contrib_mongoengine_fields api/mod_contrib_peewee api/mod_contrib_pymongo api/mod_contrib_fileadmin api/mod_model_template changelog 
resolving references...
writing... done
copying TeX support files...
done
build succeeded, 18 warnings.
```

**Comment by petrus-jvrensburg** [comment]
Okay, I'll look into it.

**Comment by petrus-jvrensburg** [comment]
Fixed in 01589c4c0fe5ad0a61202f333101580ec9772c17.
